features/step_definitions/kata.rb

```diff
@@ -83,3 +83,78 @@
   @result = node(pod.node_name).host.exec_admin(node_cmd)
   raise "No qemu process detected inside pod node" unless @result[:response].include? 'qemu'
 end
+
+# generate a cert for the kata-webhook pod which involves
+# 1. generate a cert for the webhook
+# 2. create certs secrets for k8s
+# 3. set the CABundle on the webhook registration yaml
+Given /^I install kata-webhook for the#{OPT_QUOTED} namespace$/ do | ns |
+  # ignore the admin check if it's for kata
+  unless ENV.has_key? 'USE_KATA_RUNTIME' and ENV['USE_KATA_RUNTIME']
```

Comment:
you probably want `to_bool(ENV['USE_KATA_RUNTIME']}`, no need to check key exists